SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -739,6 +767,59 @@ void FSpatialGDKEditorToolbarModule::StopSpatialServiceButtonClicked()
 	});
 }
 
+void FSpatialGDKEditorToolbarModule::ToggleSpatialDebuggerEditor()
+{
+	// Don't try and start spatial editor debugger spatial networking is disabled.
```

Comment:
nits
```suggestion
	// Don't try and show SpatialDebuggerEditor if Spatial networking is disabled.
```

Reply:
Do we need this to be an error? I think this could be a warning or even a no-op? Better yet (but only a nice-to-have and not a requirement at all) would be disabling the ToggleSpatialDebuggerEditor option is spatial networking is disabled?